tests/network/vmi_multus.go

```diff
@@ -1520,10 +1516,10 @@ func validateSRIOVSetup(virtClient kubecli.KubevirtClient, sriovResourceName str
 	return nil
 }
 
-func validatePodKubevirtResourceName(virtClient kubecli.KubevirtClient, vmiPod *k8sv1.Pod, networkName, sriovResourceName string) error {
+func validatePodKubevirtResourceName(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, networkName, sriovResourceName string) error {
```

Comment:
Following https://github.com/kubevirt/kubevirt/pull/6113#discussion_r683574579,
Lets change the name now that we no longer pass the Pod, 
maybe something like `validatePodKubevirtResourceNameByVMI`?

Reply:
yep, makes sense